### DISCUSSION TITLE
STAMPA PAGELLA GENITORI/STUDENTI

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#357] [Idee e proposte]
sarebbe possibile come in LAMPSmpschool,anche in giuaschool lasciare il libero arbitrio alla segreteria sul fatto di far stampare o meno la pagella/certificato delle competenze  ai genitori/alunni?Grazie

**Comment by trinko** [comment]
La segno come richiesta.
